upgrade/Stage_0_Prerequisites.md

```diff
@@ -48,14 +48,34 @@ after a break, always be sure that a typescript is running before proceeding.
 
    - If `ncn-m001` has internet access, then use the following commands to download and install the latest documentation.
 
-      > **Important:** The upgrade scripts expect the `docs-csm` RPM to be located at `/root/docs-csm-latest.noarch.rpm`; that is why these commands copy it there.
+      > **Important:** The upgrade scripts expect the `docs-csm` RPM to be located at `/root/docs-csm-latest.noarch.rpm`; that is why these commands copy it there.  
+      > ***NOTE:*** CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
+      Using `http_proxy` or `https_proxy` in any way other than the following examples will cause many failures in subsequent steps.
+
+      Without proxy:
 
       ```bash
       wget https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.3/noarch/docs-csm-latest.noarch.rpm \
           -O /root/docs-csm-latest.noarch.rpm &&
       rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
       ```
 
+      With https proxy:
+
+      ```bash
+      https_proxy=https://example.proxy.net:443 wget https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.3/noarch/docs-csm-latest.noarch.rpm \
+          -O /root/docs-csm-latest.noarch.rpm &&
+      rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
+      ```
+
+      With http proxy:
```

Comment:
Do we need the http proxy section here? We are giving them a specific URL for this artifact and it is an HTTPS URL. So I think the http_proxy would just be ignored in this case.